ibexOverview.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <META HTTP-EQUIV="CACHE-CONTROL" CONTENT="NO-CACHE">
+  <title>IBEX status</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+  <script src="ibexOverview.js" type="text/javascript" defer></script>
+  <link rel="shortcut icon" type="image/x-icon" href="favicon.ico"/>
+  <style>    
+    /* Set gray background color and 100% height */
+    .sidenav {
+      background-color: #f1f1f1;
+      height: 100%;
+    }
+    .btn-xl {
+	class:text-center;
+    padding: 10px 20px;
+    font-size: 20px;
+    border-radius: 10px;
+    width:20%;
+	}
+    .btn-large {
+	class:text-center;
+    padding: 10px 20px;
+    font-size: 20px;
+    border-radius: 10px;
+    width:10%;
+	}
+    /* On small screens, set height to 'auto' for the grid */
+    @media screen and (max-width: 767px) {
+      .row.content {height: auto;} 
+	
+
+    }
+  </style>
```

Comment:
General good practice is to have custom styles in a CSS file, not defined in line. We have multiple HTML files that might share styles.